packages/web/src/components/layouts/SearchResults/SearchResults.tsx

```diff
@@ -21,33 +21,48 @@ type Props = {
   packagesStats: { total: number; vulnerable: number; outdated: number };
   vulnerabilities: Record<string, ClientApi.PackageVulnerabilityResponse[]>;
   vulnerabilitiesCount: number;
+  scriptsCount: number;
+  bundleSize: number;
   keywordsList: string[];
   scanDate?: string;
   // siteFavicon: string;
 };
 
+function getReadableSizeString(sizeInBytes: number) {
+  let i = -1;
+  const byteUnits = ['kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+  do {
+    sizeInBytes = sizeInBytes / 1024;
+    i++;
+  } while (sizeInBytes > 1024);
+
+  return `${Math.max(sizeInBytes, 0.1).toFixed(1)} ${byteUnits[i]}`;
```

Comment:
Side effect: if the size is 0, the page will display `0.1 KB`, which is wrong.

Reply:
Not sure how to fix this properly, added an edge case handler